src/types.ts

```diff
@@ -103,4 +105,9 @@ export function isNullAddress(
   }
   return Object.values(NullAddresses).includes(key);
 }
+
+export function isSupportedChainVersion(obj: any): obj is TickerVersion {
```

Comment:
@JohnnyJumper I would not add functions to the `types` file. Please, move out other functions as well.